### ISSUE TITLE
English-Logo for Björn Steiger Stiftung on english version

### ISSUE THREAD

**Issue by topialla** [LabHive/LabHive#90]
on the english site, there should/could be an english version of the foundations logo. Don't know, how much work this is, though... not that high priority

Logo can be downloaded here:
https://www.dropbox.com/s/cwg5ks0xeqlbrx9/Logo%20zur%20Verwendung_en.png?dl=0

**Comment by frwms** [comment]
@Kavakuo I tried achieving that by putting the path to the images in the translations and then getting that path into the image via v-bind but without success. HTML looked correct but there were no images behind the paths. I think Vue alters the paths and/or folder of images on compile? Maybe you have a better idea.

**Comment by Kavakuo** [comment]
Yes, the path are altered by Vue. If you put the images into the `public` folder, this should work, the paths are not changed then.

**Comment by topialla** [comment]
Did that work? @frwms
